### PR TITLE
Argon

### PR DESCRIPTION
**Pull request by zakird**

### CODE REVIEW

File: proto/ct.proto [hunk 1]
```diff
@@ -131,7 +150,10 @@ enum CTServer {
     CT_SERVER_CERTIFICATETRANSPARENCY_CN_CT = 38; // ACTIVE www.certificatetransparency.cn/ct
     CT_SERVER_SHECA_CT = 42; // ACTIVE     https://ct.sheca.com
 
-    CT_SERVER_LETSENCRYPT_CT_CLICKY = 44; // clicky.ct.letsencrypt.org
+    CT_SERVER_LETSENCRYPT_CT_CLICKY = 43; // clicky.ct.letsencrypt.org
```
Comment (dadrian) [inline proto/ct.proto]: Why did this number change?

Comment (zakird) [reply proto/ct.proto]: Because it now matches the other places in the file and doesn't skip 43.

Comment (cdzombak) [reply proto/ct.proto]: Is this number persisted anywhere as an identifier for the Clicky log?

**Comment by dadrian** [comment]
After our discussion last night, it occured to me that we can actually check to see if there's any persisted data from Clicky by querying BQ. There's not. Therefore, I am comfortable changing the enum value to match / be correct. We will have to make sure we update zdb _before_ the rest of the downstream services.

**Comment by dadrian** [comment]
```
SELECT COUNT(*) FROM [steady-circuit-91417:certificates.certificates] 
WHERE ct.letsencrypt_ct_clicky.index IS NOT NULL
LIMIT 1000
```
=> 0

**Pull request merged**